gradle-scripts/version-setter.gradle

```diff
@@ -0,0 +1,32 @@
+task injectVersion {
+    description 'If the env var "TRAVIS_TAG" is set, injects the value in the ' +
+            'src/main/java/com/commercetools/sync/commons/utils/SyncSolutionInfo.java. Otherwise, if the env var is not ' +
+            'set it sets the version to the value "dev-version". Note: Should only be executed before compilation in ' +
+            'the CI tool (e.g. Travis.)'
+    doLast {
+        def versionFile = 'src/main/java/com/commercetools/sync/commons/utils/SyncSolutionInfo.java'
+        def versionFileContents = new File(versionFile).text
+        def versionPlaceholder = '#{LIB_VERSION}'
+        def libVersion = 'dev-version'
+        def isPr = System.getenv('TRAVIS_PULL_REQUEST')
+        def tagName = System.getenv('TRAVIS_TAG')
+
+        if (!versionFileContents.contains(versionPlaceholder)) {
+            throw new InvalidUserCodeException("$versionFile does not contain the placeholder: $versionPlaceholder. " +
+                    "Please make sure the file contains the placeholder, in order for the version to be injected " +
+                    "correctly.")
+        }
+
+        if (isPr.equals("false") && tagName) {
```

Comment:
use _groove goodness_ :) :
```groovy
isPr == "false"
```
http://mrhaki.blogspot.de/2009/09/groovy-goodness-check-for-object.html

Reply:
168c067